tslint/tslint_annotations.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotationLevel } = require('../annotations_levels')
+
+/**
+ * Translates the tslint severity from "ERROR" | "WARNING" | "OFF"
+ * to the "HIGH", "MEDIUM" and "LOW"
+ * @param {String} severity issues severity given by tslint
+ */
+function translateSeverity (severity) {
+  let result = ''
+  switch (severity) {
+    case 'ERROR': result = 'HIGH'; break
+    case 'WARNING': result = 'MEDIUM'; break
+    case 'OFF': result = 'LOW'
```

Comment:
It's still not clear to me what OFF severity means.

Reply:
To we fair I am not sure after spending a few hours investigating. 
I opened an issue about it: https://github.com/palantir/tslint/issues/4634